config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "guests/:id", to: "guests#show"
+  post "guests/:id/add_room", to: "guests#add_room"
```

Comment:
As mentioned in previous comments, a more RESTful controller organization could be using a joins controller like so:
`post "/guests/:id/rooms", to: "guest_rooms#create"`